src/main/resources/db/changelog/changes/9-create-shopping-cart-table.yaml

```diff
@@ -0,0 +1,31 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-shopping-cart-table
+      author: daniil
+      changes:
+        - createTable:
+            tableName: shopping_carts
+            columns:
+              - column:
+                  name: id
+                  type: BIGINT
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+              - column:
+                  name: user_id
```

Comment:
remove

Reply:
dont understand what exactly needs to be deleted

Reply:
please read the comment above. Remove FK constraint